src/components/SEO/index.js

```diff
@@ -64,21 +130,26 @@ function SEO(props) {
         },
         {
           name: `twitter:title`,
-          content: title,
+          content: seo.title,
         },
         {
           name: `twitter:description`,
-          content: metaDescription,
+          content: seo.description,
+        },
+        {
+          name: `twitter:image`,
+          content: seo.image,
         },
       ].concat(meta)}
     >
-      <title>{pageTitle}</title>
+      <title>{pageDisplayTitle}</title>
+      <link rel="canonical" href={seo.url} />
     </Helmet>
   );
 }
 
 SEO.defaultProps = {
-  lang: `en-GB`,
+  lang: undefined,
```

Comment:
Should this be undefined? Don't we have a site language set somewhere?

Reply:
Confusingly, this means it actually uses the `siteConfig` fallback. Maybe we ought to document that:
```suggestion
  // If this prop is `undefined`, `siteConfig.language` will be used as a fallback.
  lang: undefined,
```